src/plugins/kibana/server/routes/api/ingest/register_processors.js

```diff
@@ -0,0 +1,24 @@
+import _ from 'lodash';
+import handleESError from '../../../lib/handle_es_error';
+import handleResponse from '../../../lib/process_es_ingest_processors_response';
+import { keysToCamelCaseShallow, keysToSnakeCaseShallow } from '../../../../common/lib/case_conversion';
+
+export function registerProcessors(server) {
+  server.route({
+    path: '/api/kibana/ingest/processors',
```

Comment:
We're establishing a bad pattern here with simulate and now this. This url could conflict with the ingest config endpoints (which will eventually support GET) if the user creates an index called `processors`.

We either need to come up with different urls for this endpoint and simulate, or we should move the existing ingest API endpoints to something like `/api/kibana/ingest/config/{id}`. What do you think?